tests/testflows/rbac/tests/syntax/alter_row_policy.py

```diff
@@ -0,0 +1,244 @@
+                    node.query("ALTER ROW POLICY policy4 ON default.foo AS PERMISSIVE")
+
+        with Scenario("I alter row policy to restrictive", flags=TE, requirements=[
+                RQ_SRS_006_RBAC_RowPolicy_Alter_Access_Restrictive("1.0"),
+                RQ_SRS_006_RBAC_RowPolicy_Alter_On("1.0")]):
+            with cleanup("policy5"):
+                with When("I alter row policy as restrictive"):
+                    node.query("ALTER ROW POLICY policy5 ON default.foo AS RESTRICTIVE")
+
+        with Scenario("I alter row policy for select", flags=TE, requirements=[
+                RQ_SRS_006_RBAC_RowPolicy_Alter_ForSelect("1.0"),
+                RQ_SRS_006_RBAC_RowPolicy_Alter_On("1.0")]):
+            with cleanup("policy6"):
+                with When("I alter row policy using for select"):
+                    node.query("ALTER ROW POLICY policy6 ON default.foo FOR SELECT USING x > 10")
+
+        with Scenario("I alter row policy using condition", flags=TE, requirements=[
+                RQ_SRS_006_RBAC_RowPolicy_Alter_Condition("1.0"),
+                RQ_SRS_006_RBAC_RowPolicy_Alter_On("1.0")]):
+            with cleanup("policy6"):
+                with When("I alter row policy wtih condition"):
+                    node.query("ALTER ROW POLICY policy6 ON default.foo USING x > 10")
+
+        with Scenario("I alter row policy using condition none", flags=TE, requirements=[
+                RQ_SRS_006_RBAC_RowPolicy_Alter_Condition_None("1.0"),
+                RQ_SRS_006_RBAC_RowPolicy_Alter_On("1.0")]):
+            with cleanup("policy7"):
+                with When("I alter row policy using no condition"):
+                    node.query("ALTER ROW POLICY policy7 ON default.foo USING NONE")
+
+        with Scenario("I alter row policy to one role", flags=TE, requirements=[
+                RQ_SRS_006_RBAC_RowPolicy_Alter_Assignment("1.0"),
+                RQ_SRS_006_RBAC_RowPolicy_Alter_On("1.0")]):
+            with cleanup("policy8"):
+                with When("I alter row policy to a role"):
+                    node.query("ALTER ROW POLICY policy8 ON default.foo TO role0")
+
+        with Scenario("I alter row policy to assign to role that does not exist, throws exception", flags=TE, requirements=[
+                RQ_SRS_006_RBAC_RowPolicy_Alter_Assignment("1.0")]):
+            role = "role2"
+            with cleanup("policy8a"):
+                with Given(f"I drop {role} if it exists"):
+                    node.query(f"DROP ROLE IF EXISTS {role}")
+                with Then(f"I alter a row policy, assign to role {role}, which does not exist"):
+                    exitcode, message = errors.role_not_found_in_disk(name=role)
+                    node.query(f"ALTER ROW POLICY policy8a ON default.foo TO {role}", exitcode=exitcode, message=message)
+            del role
+
+        with Scenario("I alter row policy to assign to all excpet role that does not exist, throws exception", flags=TE, requirements=[
```

Comment:
What do the flags `TE` mean?

Reply:
It means that we are marking this test as `test-to-end` (https://testflows.com/handbook/#TE) and it means that
the test flow will not abort if this test does not pass. By default test flow would be aborted on fail (`first-fail`).